hw/ip/alert_handler/data/alert_handler.hjson

```diff
@@ -334,7 +334,7 @@
     },
     { multireg: { name:     "LOC_ALERT_EN",
                   desc:     '''Enable register for the aggregated local alerts "alert
-                  pingfail" (0), "escalation pingfail" (1), "alert integfail" (2) and "escalation integfail" (3).
+                  pingfail" (0), "escalation pingfail" (1), "alert integfail" (2), "escalation integfail" (3), and "bus integrity failure (4)".
```

Comment:
nit: the rest of the failures are formatted as this:
and "bus integrity failure" (4).

Reply:
same below :)

Reply:
woops, thanks. I'll fix this in a follow-up where I am going to add the shadow register internal alerts. otherwise we have to wait for CI again.